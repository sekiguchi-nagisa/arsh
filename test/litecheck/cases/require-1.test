#!/bin/sh
echo 12345
echo ABCDE
echo abcde
echo @@@@@
echo 222 1>&2
false
# REQUIRE: command -v /bin/sh
# CHECK: 12345
# CHECK: ABCDE
# CHECK: abde
# CHECK: @@@@@
# STATUS: 1