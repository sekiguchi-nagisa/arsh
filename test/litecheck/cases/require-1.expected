CHECK on line 9,
3c3
< abde
---
> abcde
stdout is mismatched